src/routes/posts.js

```diff
@@ -105,6 +105,66 @@ const fetchPosts = async (req, res, next) => {
     }
 }
 
+// ASSIGN TAG TO AN ARTICLE
+
+const assignTagToPost = async (req, res) => {
+    try {
+        const { postId } = req.params // post id;
+        const { tagId } = req.body
+
+        const postsTags = await db.query(
+            'INSERT INTO posts_tags ("postId","tagId") SELECT $1,$2 WHERE NOT EXISTS (SELECT * FROM posts_tags WHERE "postId" = $1 AND "tagId" = $2) RETURNING *',
+            [postId, tagId]
+        )
+
+        if (!postsTags.rows[0]) {
```

Comment:
@Yellow-May  can you shed some more light on how this works for us to know tag is already assigned?

Reply:
Okay, so if a tag is assigned to a post, it returns the the post details ONLY if the the combination of the "postId" and "tagId" do not exist in the table

So if it already exists, it will not return anything so we can tell that the tag has already been assigned to that post

Reply:
cool, thanks